src/common/options/rbd.yaml.in

```diff
@@ -387,10 +387,10 @@ options:
   default: false
   services:
   - rbd
-- name: rbd_blkin_trace_all
```

Comment:
is it backwward compatible to remove a conf option?
what will happen if the old one is being set somewhere?